DocumentedExamples/SeaIce_Obs_Model_Compare.ipynb

```diff
@@ -0,0 +1,5185 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #9.        #Divide by 1000 to get km2
</pre><p>what are original units? would need to divide by 10^6 to convert from m^2 to km^2</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/COSIMA/cosima-recipes/pull/311/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='27'/>

Reply:
<p>decimetres !</p>

Reply:
<p>weird! as in 10cm? Would the area conversion factor then be 10^8?</p>

Reply:
Oh sorry - 10m. i.e. the divide by 1000 is correct

Reply:
<p>ok, decametres then. But the area factor would then be 10^4, right?</p>

Reply:
<p>Ok. Im out. I dont have any explanation beyong this works and other people do it. (e.g. <a href="https://github.com/ethan-campbell/Weddell_polynya_paper/blob/387a597d832179d7605ca9c7cc5634a43bfb62cb/load_product.py#L2557" target="_blank">https://github.com/ethan-campbell/Weddell_polynya_paper/blob/387a597d832179d7605ca9c7cc5634a43bfb62cb/load_product.py#L2557</a>)</p><p> </p>

Reply:
<p>hm, that's very odd. sqrt(1000) is irrational, so it's hard to see how this could be a correct area conversion factor no matter what the linear units are...</p>

Reply:
<p>maybe they simply saved 1000*area_in_km2 in the file?</p>

Reply:
<p>Oh I found it: <a href="https://web.archive.org/web/20170817210544/http://nsidc.org/data/polar-stereo/tools_geo_pixel.html#pixel_area" target="_blank">https://web.archive.org/web/20170817210544/http://nsidc.org/data/polar-stereo/tools_geo_pixel.html#pixel_area</a></p><p> </p><p>NSIDC are pushing people to use NSIDC-0771 now for the cell areas, but its a pain to access because you need an account (so wouldn't run seamlessly in a workbook for multiple users). I am guessing that is why this explanation is only on the wayback machine. </p><p> </p><p>"The arrays are in binary format and are stored as 4-byte integers scaled by 1000 (divide by 1000 to get square km)."</p>

Reply:
<p>Aha! Thanks for going down that rabbithole.</p>